easy_install/script.sh

```diff
@@ -18,6 +18,7 @@ OVERWRITE=${OVERWRITE=y}
 declare -A INSTALL_TBL
 INSTALL_TBL[41]="varnish-plus varnish-plus-vmods-extra varnish-plus-addon-ssl varnish-agent varnish-plus-ha varnish-broadcaster"
 INSTALL_TBL[60]="varnish-plus varnish-plus-vmods-extra varnish-plus-addon-ssl varnish-agent varnish-plus-ha varnish-broadcaster"
+INSTALL_TBL[controller]="varnish-controller-agent varnish-controller-brainz varnish-controller-api-gw varnish-controller-nats varnish-controller-cli"
```

Comment:
aren't we going to version this repo?

Reply:
I have no idea. I guess that is @stone's area

Reply:
Don't know what this is about, sorry.